src/ngraph/frontend/onnx_import/utils/common.cpp

```diff
@@ -79,6 +80,17 @@ namespace ngraph
                 return new_axes;
             }
 
+            ngraph::NodeVector get_outputs(const std::shared_ptr<ngraph::Node>& node,
+                                           int64_t outputs_number)
+            {
+                ngraph::NodeVector outputs(outputs_number);
+                for (int i = 0; i < outputs_number; ++i)
+                {
+                    outputs[i] = std::make_shared<ngraph::op::GetOutputElement>(node, i);
```

Comment:
`outputs[i] = Output(node, i).as_single_output_node()` re-uses existing `GetOutputElement` nodes and only creates `GetOutputElement` if the output needs one.
There is also `as_node_vector(node.outputs())` which would do everything this method does without the need for `outputs_number`.

Reply:
In `as_node_vector` is called `as_single_output_node` which requires `node->get_output_size() == 1'` which is not true for some tests (such as `model_split_equal_parts_default`).
Similar problem occurs for `as_single_output_node`. It also requires that output node must produce exactly one value.

Reply:
@diyessi In `as_node_vector` is called `as_single_output_node` which requires `node->get_output_size() == 1'` which is not true for some tests (such as `model_split_equal_parts_default`). Assert in `check_single_output_arg` fails.
Similar problem occurs for version `outputs[i] = node->get_output_as_single_output_node(i)` - assert `check_single_output_args` fails.
I am not sure how to deal with it.

Reply:
It should be returning the real node itself for single-output nodes, and GetOutputElement(real node, i) for the other nodes.

Reply:
@diyessi Thanks. I introduced this check.